### DISCUSSION TITLE
Microchip claims DxCore provides full Arduino Support

### DISCUSSION THREAD

**Discussion by MX682X** [SpenceKonde/DxCore#289] [Show and tell]
[https://iot.microchip.com/docs/arduino/start](https://iot.microchip.com/docs/arduino/start)

Congrats, Spence, even the manufacturer of this chips relies in it's advertising on (mainly) your work.

**Comment by obdevel** [comment]
And a shout out from Lady Ada in a recent video: https://www.youtube.com/watch?v=EA0B8s6Thr4
